### ISSUE TITLE
TypeError: expected author to be a string

### ISSUE THREAD

**Issue by o2dazone** [brunocodutra/webapp-webpack-plugin#17]
Came across this funny error when using your package (thanks btw! Webpack 4 support woohoo)

`TypeError: expected author to be a string`

In our `package.json`, we have `author` as an object, with name, url and email keys. [Looking at the spec](https://docs.npmjs.com/files/package.json#people-fields-author-contributors), this seems to be valid. But it dies when trying to guess the developer names from the package.json.

If I get some time I can submit a PR, or if this is already on your radar, then I wont :)

Anyway, thanks for the plugin! 👍

**Comment by brunocodutra** [comment]
Great find, thanks for reporting!

This is one of those bugs for which writing regression tests is a lot more work than writing the actual fix. If you manage to find time to submit a PR with a fix while I work on tests it would be more than welcome of course!

As a workaround in the meantime, you can prevent it from attempting to parse the author field in `package.json` by manually adding `developerName` and `developerURL` to `options.favicons`:

```.js
new WebappWebpackPlugin({
    logo: 'logo.png',
    favicons {
      ...
      developerName: 'John Doe',
      developerURL: 'john.doe.com',
      ...
    }
  })
```

**Comment by brunocodutra** [comment]
Should be fixed in [1.1.2.](https://github.com/brunocodutra/webapp-webpack-plugin/releases/tag/v1.1.2)

Feel free to reopen in case you have further issues.
